src/login/Login.js

```diff
@@ -24,7 +24,7 @@ export default class Login extends React.Component {
         this.errors = json.text
         otpLength = json.otpLength;
       } catch (e) {
-        this.errors = `could not pass error json: ${e}`;
+        /* */
```

Comment:
Shouldn't this set at least a generic error message? What is `this.errors` used for?

Reply:
this.errors is errors passed from passport. They can be a string "invalid credentials" or a JSON string with an error

Reply:
I'm not comfortable with an empty catch block - lint should complain here. In which cases could an error be thrown and wouldn't we want to show a specific error message in that case?

Reply:
I’ll rethink this

Reply:
If it makes sense maybe something like:

```suggestion
        this.errors = this.errors ?? `Error: ${JSON.stringify(e)}`;
```